indico/modules/rb_new/schemas.py

```diff
@@ -158,10 +158,11 @@ class BlockingSchema(mm.ModelSchema):
     blocked_rooms = Nested(BlockedRoomSchema, many=True)
     allowed = Nested(BlockingPrincipalSchema, many=True)
     can_edit = Function(lambda blocking: blocking.can_be_modified(session.user))
+    created_by = Nested(UserSchema, attribute='created_by_user', only=('full_name',))
```

Comment:
if you use a string instead of a tuple you get rid of the nesting so `created_by` is a string. imho that'd be cleaner here since then you can just use `blocking.createdBy` in the frontend

Reply:
Good to know, quite useful.